extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -176,12 +176,15 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 	private _currentState: State;
 	private _gatheringInformationError: string | undefined;
 
-	private _skuRecommendations: SKURecommendations | undefined;
+	private _skuRecommendations: SKURecommendations | undefined;		// TO-DO: old, can this be removed?
```

Comment:
this can be removed

Reply:
Got it, just wasn't sure if it served any purpose so I left it in